api/HTMLDialogElement.json

```diff
@@ -79,28 +89,38 @@
             "edge": {
               "version_added": "79"
             },
-            "firefox": {
-              "version_added": "78",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.dialog_element.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
-            },
-            "firefox_android": {
-              "version_added": "79",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "dom.dialog_element.enabled",
-                  "value_to_set": "true"
-                }
-              ],
-              "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
-            },
+            "firefox": [
+              {
+                "version_added": "98"
+              },
+              {
+                "version_added": "78",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.dialog_element.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "98"
+              },
+              {
+                "version_added": "79",
+                "flags": [
+                  {
+                    "type": "preference",
+                    "name": "dom.dialog_element.enabled",
+                    "value_to_set": "true"
+                  }
+                ],
+                "notes": "See <a href='https://bugzil.la/840640'>bug 840640</a>."
+              }
```

Comment:
This support statement looks pretty suspect in general. I'd remove the whole thing and only show the version_added 98 statement.